iwf-sdk.yaml

```diff
@@ -1208,13 +1208,10 @@ components:
       type: object
       required:
         - commandId
-      # either firingUnixTimestampSeconds or durationSeconds must be provided
+        - durationSeconds
```

Comment:
I saw that we removed `firingUnixTimestampSeconds` from the required props in https://github.com/indeedeng/iwf-idl/commit/f3d0dba7a380d0a459a1b6549bd2ee8df5f66c0f to allow both parameters to be optional. 

Because we're now removing `firingUnixTimestampSeconds` from the SDK I'm guessing `durationSeconds` would become required

Reply:
good catch!  that makes sense to me as well, since the TimerCommand must have a value, but I'll let @longquanzheng confirm

Reply:
Yes that is correct 👍